Patch configure.ac to not use its own CFLAGS with our LDFLAGS when checking
time_t size / size of int, this results in a compilation error building the test.
diff -durN vice-3.6.0-orig/configure.ac vice-3.6.0/configure.ac
--- vice-3.6.0-orig/configure.ac	2021-12-24 20:27:40.000000000 +0100
+++ vice-3.6.0/configure.ac	2021-12-26 17:02:12.549832428 +0100
@@ -528,7 +528,6 @@
 
 
 old_CFLAGS="$CFLAGS"
-CFLAGS="$VICE_CFLAGS"
 dnl Do these checks before any Gtk3 flags are added
 AC_MSG_CHECKING(for time_t in time.h)
 AC_CACHE_VAL(bu_cv_decl_time_t_time_h,
diff -up vice-3.6.1/src/resid-dtv/configure.in~ vice-3.6.1/src/resid-dtv/configure.in
--- vice-3.6.1/src/resid-dtv/configure.in~	2022-01-24 01:13:25.000000000 +0100
+++ vice-3.6.1/src/resid-dtv/configure.in	2022-02-08 11:38:31.075122918 +0100
@@ -40,7 +40,7 @@ AC_PROG_CXX([c++ g++ clang++])
 dnl Set CXXFLAGS for g++. Use -fno-exceptions if supported.
 if test "$GXX" = yes; then
   if test "$ac_test_CXXFLAGS" != set; then
-    CXXFLAGS="-g -Wall -O2 -funroll-loops -fomit-frame-pointer -fno-exceptions"
+    CXXFLAGS="$CXXFLAGS -g -Wall -O2 -funroll-loops -fomit-frame-pointer -fno-exceptions"
     AC_MSG_CHECKING([whether the C++ compiler ($CXX $CXXFLAGS $LDFLAGS) works])
     AC_TRY_COMPILE([],
                    [int test;],
